utils/format/.gitlint

```diff
@@ -11,6 +11,9 @@ ignore-squash-commits=false
 # Enable debug mode (prints more output). Disabled by default
 debug = false
 
+# enable conventional commits
+contrib = contrib-title-conventional-commits,CC1
```

Comment:
I think we actually want CT1, not CC1

Reply:
https://jorisroovers.com/gitlint/latest/rules/contrib_rules/#ct1-contrib-title-conventional-commits

Reply:
ty